README.md

```diff
@@ -2,7 +2,8 @@ Basic economy for the fabric modloader using diamonds as currency. Uses the "/di
 
 - balance [Optional: player] - tells user how much money the player has
```

Comment:
I not aware of amount exchanged at instance the balance is calculated is subject to change values